### PR TITLE
1.7 maintenance branch provides Java EE API based version only

### PR DESCRIPTION
**Pull request by ropalka**
https://github.com/jbossas/jboss-invocation/issues/39

### CODE REVIEW

**Comment by ropalka** [comment]
@chengfang please review

File: pom.xml [hunk 1]
```diff
@@ -48,158 +48,112 @@
...
+        </dependency>
+        <dependency>
+            <groupId>org.jboss.spec.javax.interceptor</groupId>
+            <artifactId>jboss-interceptors-api_1.2_spec</artifactId>
+            <version>${version.javax.interceptor}</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.jboss.spec.javax.transaction</groupId>
+            <artifactId>jboss-transaction-api_1.2_spec</artifactId>
```
Comment (chengfang) [inline pom.xml:128]: can we use `jboss-transaction-api_1.3_spec 2.0.0.Final`, which is used by WF and EAP 7 (https://github.com/wildfly/wildfly/blob/main/pom.xml#L558)

Comment (ropalka) [reply pom.xml]: Fixed @chengfang - good catch!

**Review by chengfang — APPROVED** [review]

**Comment by ropalka** [comment]
Thanks for review @chengfang !

**Pull request merged**